### PR TITLE
Bump golang.org/x/net to v0.8.0 in v0.30.x

### PR DESCRIPTION
**Pull request by sethiyash**
- Bump golang.org/x/net to v0.8.0 in v0.30.x

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -16,59 +16,61 @@ require (
...
+	k8s.io/apimachinery v0.24.3
 	k8s.io/code-generator v0.17.2
-	sigs.k8s.io/yaml v1.2.0
+	sigs.k8s.io/yaml v1.3.0
 )
 
 require (
-	github.com/google/go-containerregistry v0.10.0
-	github.com/phayes/freeport v0.0.0-20220201140144-74d24b5ae9f5
+	github.com/google/go-containerregistry v0.13.0
```
Comment (joaopapereira) [inline go.mod]: nit: Do you mind moving all the indirect dependencies to one `require` block and all the direct one to a different block?

**Review by joaopapereira — CHANGES_REQUESTED** [review]
Going to request the change just for consistency's sake. Apart from that it looks good the PR

File: go.mod (hunk 1 above)
Comment (sethiyash) [reply go.mod]: Done. Moved all direct and indirect dependencies into separate blocks.

**Review by joaopapereira — APPROVED** [review]
LGTM

**Pull request merged**